packages/paste-website/src/pages/articles/2021-07-21-pastes-path-to-a-transparent-package-categorization-system.mdx

```diff
@@ -0,0 +1,121 @@
+---
+title: Paste's Path to a Transparent Package Categorization System
+description: A brief outline detailing how Twilio's Design System previously categorized component and pattern packages and what sparked a change.
+slug: /articles/2021-07-21-pastes-path-to-a-transparent-package-categorization-system/
+date: 2021-07-21
+author: Nora Krantz
+avatar: https://avatars.githubusercontent.com/u/75342690?v=4
+author_link: https://www.linkedin.com/in/norakrantz/
+status: published
+---
+
+import {graphql} from 'gatsby';
+import Img from 'gatsby-image';
+import {Box} from '@twilio-paste/box';
+
+export const pageQuery = graphql`
+  {
+    mdx(fields: {slug: {eq: "/articles/2021-07-21-pastes-path-to-a-transparent-package-categorization-system/"}}) {
+      fileAbsolutePath
+      frontmatter {
+        slug
+        title
+        description
+        date(formatString: "MMMM D, YYYY")
+        machineDate: date
+        author
+        avatar
+        author_link
+      }
+      headings {
+        depth
+        value
+      }
+    }
+    overview1: file(
+      sourceInstanceName: {eq: "assets"}
+      relativePath: {eq: "images/articles/pastes-path-to-a-transparent-package-categorization-system/overview-1.png"}
+    ) {
+      childImageSharp {
+        fluid(maxWidth: 500) {
+          ...GatsbyImageSharpFluid
+        }
+      }
+    }
+    overview2: file(
+      sourceInstanceName: {eq: "assets"}
+      relativePath: {eq: "images/articles/pastes-path-to-a-transparent-package-categorization-system/overview-2.png"}
+    ) {
+      childImageSharp {
+        fluid(maxWidth: 500) {
+          ...GatsbyImageSharpFluid
+        }
+      }
+    }
+  }
+`;
+
+<ArticleHeader
+  title={props.data.mdx.frontmatter.title}
+  description={props.data.mdx.frontmatter.description}
+  date={props.data.mdx.frontmatter.date}
+  machineDate={props.data.mdx.frontmatter.machineDate}
+  author={props.data.mdx.frontmatter.author}
+  avatar={props.data.mdx.frontmatter.avatar}
+  authorLink={props.data.mdx.frontmatter.author_link}
+/>
+
+---
+
+<contentwrapper>
+
+<ArticleContent>
+
+_When integrating a library into a project, you’ll likely want to ensure that it’s a mature and well-maintained library that won’t be deprecated a month down the line. If your project is a customer-facing platform and the library maintains your frontend components, all the more reason to have trust. Transparency between the library and the consumer is integral to building that trust._
```

Comment:
vvv small nit, the italics create nice visual hierarchy at the top but not sure if it's a quote or summary of something later in the post

Reply:
removed italics!